cluster/node-pools/master-default/userdata.yaml

```diff
@@ -666,7 +666,7 @@ write_files:
         - name: kube-scheduler
           image: nonexistent.zalan.do/teapot/{{if eq .Cluster.ConfigItems.kubernetes_scheduler_image "zalando" }}kube-scheduler-internal{{else}}kube-scheduler{{end}}:fixed
           args:
-          - --kubeconfig=/etc/kubernetes/scheduler-kubeconfig
```

Comment:
Why can this be removed? 🤔

Reply:
This flag became part of the config file itself: https://github.com/zalando-incubator/kubernetes-on-aws/pull/7628/files#diff-fb712e77981d2e840a03b53964fd2fb37d9cd137e35b2db4d1dbb36f5170fb44R869

There are a handful of other potentially interesting knobs we can turn in the scheduler config. Full reference is here: https://kubernetes.io/docs/reference/config-api/kube-scheduler-config.v1/#kubescheduler-config-k8s-io-v1-KubeSchedulerConfiguration

![image](https://github.com/zalando-incubator/kubernetes-on-aws/assets/36162/9838fb32-724f-4c89-995c-a33e6d46e203)

Reply:
Ah got it, missed that it was defined below :+1: